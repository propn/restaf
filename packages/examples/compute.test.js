let testFunctions = require("./testFunctions");

test("computeService: basic", async () => {
  console.log(testFunctions.computeDS);
  let l = await testFunctions.computeDS();
  expect(l).toMatchSnapshot();
});
