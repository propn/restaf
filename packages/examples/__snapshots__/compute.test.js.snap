// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`computeService: basic 1`] = `
Array [
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    1,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    2,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    3,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    4,
    11,
  ],
  Array [
    10,
    20,
    30,
    40,
    50,
    60,
    70,
    80,
    90,
    100,
    5,
    11,
  ],
]
`;